src/index.js

```diff
@@ -12,11 +13,11 @@ import { createLogger } from 'redux-logger';
 import promise from 'redux-promise-middleware';
 import createDebounce from 'redux-debounced';
 import * as Sentry from '@sentry/browser';
+import rovercodeDefaultLight from '@/themes/rovercodeDefaultLight';
```

Comment:
I think you can put an `index.js` in `themes/` exporting each of the themes so you can do `import { rovercodeDefaultLight } from '@/themes;` and we can import multiple in the future in one import

Reply:
Hm, that's giving me: `"export 'default' (imported as 'light') was not found in '@/themes'`

Reply:
Using `export const light = createMuiTheme({` in src/themes/index.js and `import light from '@/themes`

Reply:
I think in `index.js` it would be something like this:
```
import light from light;
export default {
  light,
}
```

Reply:
ah, thanks. Still coming up to speed on js modules.

Reply:
Landed on this strategy: https://alligator.io/react/index-js-public-interfaces/